custom-routes-vis/src/main/webapp/script.js

```diff
@@ -281,15 +290,44 @@ function createRouteFromCoordinates(routeCoordinates, routeNum, totalDuration,
  * @param {num} routeNum The index of the selected route in the routes array.
  * @param {num} totalDuration The duration of the route in seconds.
  * @param {num} totalDistance The distance of the route in meters.
+ * TODO(carmenbenitez): Add in the route token returned from the Routes Preferred API.
  */
 function selectRouteDisplayDetails(routeNum, totalDuration, totalDistance) {
-  displayedRoutes[selectedRouteNum].setOptions({ strokeOpacity: 0.5, })
+  displayedRoutes[selectedRouteNum].setOptions({ strokeOpacity: 0.3, })
   selectedRouteNum = routeNum;
   displayedRoutes[routeNum].setOptions({ strokeOpacity: 1.0, });
 
   let routeInfoElement = document.getElementById('route-info');
-  routeInfoElement.innerText = 'Selected Route Info:\n' 
-      + 'Duration: ' + totalDuration + ' seconds\n'
-      + 'Distance: ' + totalDistance + ' meters\n'
-      + 'Route Token: ';
+  routeInfoElement.innerText = 'Selected Route Info:' +
+      '\nDuration: ' + formatDuration(totalDuration) +
+      '\nDistance: ' + formatDistance(totalDistance) +
+      '\nRoute Token: ';
+}
+
+/** 
+ * Formats the duration to be of the form "xx h xx min xx s."
+ * @param {num} durationSec The duration to be formatted in seconds.
```

Comment:
Add information one what you are returning.

Reply:
Sorry I meant add in an `@return {type} description` line. I think you misunderstood me.

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments